app/main.js

```diff
@@ -17,9 +17,9 @@ let aboutWin = null
 let settingsWin = null
 let finishMicrobreakTimer
 let finishBreakTimer
-let resumeBreaksTimer
 let settings
-let isPaused = false
+let toolTipUpdater
```

Comment:
Linter doesn't like this. The reference is unused right now but I was just being wary of setting an interval timer without having a reference to clear it with later. Admittedly I can't think of what scenario the tooltip updater interval would need to be cancelled though